tests/integration-tests/tests/iam/test_iam/test_iam_resource_prefix/pcluster.config.yaml

```diff
@@ -0,0 +1,49 @@
+Image:
+  Os: {{ os }}
+Iam:
+  ResourcePrefix: {{ iam_resource_prefix}}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    S3Access:
```

Comment:
Here and below: why does the cluster need access to a S3 bucket?

Reply:
I need this as S3 bucket has a few Policies. As you suggested I will test them for Updates to check the updated policies.

Reply:
Got it, thanks